### PR TITLE
fix: remove files property from package.json

### PR DESCRIPTION
**Pull request by Eladsafespace**
Removes the files property from package.json to resolve vsix packaging conflict with .vscodeignore

----

> [!IMPORTANT]
> Removes `files` property from `package.json` to resolve vsix packaging conflict with `.vscodeignore`.
> 
>   - **Behavior**:
>     - Removes `files` property from `package.json` to resolve vsix packaging conflict with `.vscodeignore`.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=RooVetGit%2FRoo-Cline&utm_source=github&utm_medium=referral)<sup> for 180e135648a3b388f16cd309f921e6f192b91c88. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

**Comment by saoudrizwan** [comment]
@mrubens @ColemanRoo Hi, I can't create an issue so I'm commenting here. Can you please change the icon for your extension on VS Marketplace? It's too similar to Cline's icon and is confusing to anyone searching for the extension. Thank you

**Comment by mrubens** [comment]
> @mrubens @ColemanRoo Hi, I can't create an issue so I'm commenting here. Can you please change the icon for your extension on VS Marketplace? It's too similar to Cline's icon and is confusing to anyone searching for the extension. Thank you

Will do!

File: package.json [hunk 1]
```diff
@@ -1,190 +1,189 @@
 {
-  "name": "roo-cline",
-  "displayName": "Roo Cline",
-  "description": "Autonomous coding agent right in your IDE, capable of creating/editing files, running commands, using the browser, and more with your permission every step of the way.",
-  "publisher": "RooVeterinaryInc",
-  "version": "2.1.6",
-  "files": [
-    "bin/roo-cline-2.1.6.vsix",
-    "assets/icons/icon_Roo.png"
-  ],
```
Comment (mrubens) [inline package.json:10]: I'll let @ColemanRoo speak to this one - I'm not that familiar with the packaging setup

File: package.json [hunk 2]
```diff
@@ -1,190 +1,189 @@
...
+		"@typescript-eslint/parser": "^7.11.0",
+		"@vscode/test-cli": "^0.0.9",
+		"@vscode/test-electron": "^2.4.0",
+		"ajv": "^8.17.1",
+		"esbuild": "^0.24.0",
+		"eslint": "^8.57.0",
+		"jest": "^29.7.0",
+		"npm-run-all": "^4.1.5",
+		"react-scripts": "^5.0.1",
+		"rewire": "^7.0.0",
```
Comment (mrubens) [inline package.json:152]: Are these changes necessary?

File: package.json (hunk 1 above)
Comment (ColemanRoo) [reply package.json:10]: I don't see any reason why these can't be removed, they were used when we were packaging this for install from an NPM repository. 
It was on my list to remove them.

**Comment by ColemanRoo** [comment]
The number of lines changed in the package.json files seems too big for just removing that files list, could you update your PR to just remove the files list please?

**Comment by mrubens** [comment]
Closing this now
